pages/common/docker-save.md

```diff
@@ -0,0 +1,24 @@
+# docker save
+
+> Export one or more docker images to archive.
+> More information: <https://docs.docker.com/engine/reference/commandline/save/>.
+
+- Save the image by redirecting output to a file:
+
+`docker save {{image}} > {{path/to/file}}`
```

Comment:
Does the filename generally have a specific extension? If so, we can indicate that here by using `path/to/file.foo.bar` (this is only an example of course).